src/diffusers/models/model_loading_utils.py

```diff
@@ -520,3 +526,64 @@ def load_gguf_checkpoint(gguf_checkpoint_path, return_tensors=False):
         parsed_parameters[name] = GGUFParameter(weights, quant_type=quant_type) if is_gguf_quant else weights
 
     return parsed_parameters
+
+
+def _find_mismatched_keys(
```

Comment:
Taken out of here:
https://github.com/huggingface/diffusers/blob/9f4d997d8ffabe455c4f743139d3b5d5039339da/src/diffusers/models/modeling_utils.py#L1509